alert-policies/dbt-cloud/dbt-failed-test-row.yml

```diff
@@ -0,0 +1,70 @@
+# Name of the alert
+name: Dbt Failed Test Row 
+
+# Description and details
+description: |+
```

Comment:
This description should be less than 240 characters

Reply:
Updated